lib/index.js

```diff
@@ -118,16 +118,12 @@ module.exports = {
     });
 
 
-    const corsMap = {};
+    const allowedMethods = new Set(['OPTIONS']);
+    const extraHeaders = new Set();
     keys(routes).forEach((routePath) => {
-      corsMap[routePath] = { shouldSet: true, extraHeaders: new Set(), allowedMethods: new Set(['OPTIONS']) };
       routes[routePath].forEach((action) => {
-        if (action.method === 'opts') { corsMap[routePath].shouldSet = false; }
```

Comment:
Removed the opts protection because of how we're now setting the options for everything on the base route.

Although we currently aren't doing it - this will cause explicit OPTIONS endpoints to just add OPTS as an allowed method which is bleh - fix inc for that